frontend/public/components/utils/_name-value-editor.scss

```diff
@@ -7,18 +7,44 @@
   opacity: 0
 }
 
-.pairs-list__field {
+.pairs-list__name-field {
+  padding-top: 5px;
   padding-right: 0;
 }
 
+.pairs-list__value-field {
```

Comment:
this can be grouped with .pairs-list_name-field above.

e.g., 
```
.pairs-list__name-field,
.pairs-list__value-field {
```

Reply:
Got it.